DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -984,7 +979,7 @@ public string GetUserFriendlyTypeName(Type t)
 
         #region Helpers
         /// <summary>
-        /// Gets the configuration-specific string comparer. This returns <see cref="StringComparer.Ordinal"/> or <see cref="StringComparer.OrdinalIgnoreCase"/>, 
+        /// Gets the configuration-specific string comparer. This returns <see cref="StringComparer.Ordinal"/> or <see cref="StringComparer.OrdinalIgnoreCase"/>,
```

Comment:
Whitespace removal is accidental, I'll return it back if it's intended to be there.